ext/bz2/bz2_filter.c

```diff
@@ -123,6 +123,7 @@ static php_stream_filter_status_t php_bz2_decompress_filter(
 				}
 			} else if (status != BZ_OK) {
 				/* Something bad happened */
+				php_error_docref(NULL, E_NOTICE, "bzip2 decompression failed");
```

Comment:
I was hoping libbz2 has a function to get an error message from the status, but that doesn't seem the case. As we probably don't want to hardcode these on our side, I guess this is fine...

Reply:
Indeed; `BZ2_bzerror()` only supports `BZFILE`, not `bz_stream`.